Rewriting vwii-modding.md (#192)
hacks-guide/Guide-WiiU: docs/vwii/vwii-modding.md

---
This page will guide you through the process of modding your vWii.

?> **Notice**
    Your SD Card will need to be formatted as FAT32. If your SD Card is not formatted to FAT32, use [GUIFormat](http://ridgecrop.co.uk/index.htm?guiformat.htm) with 32k (32768) Allocation unit size to format it. **Do not** label the SD Card as `wiiu` or it will cause issues with homebrew.
    
?> If you have hacked your Wii U in the past, you can use the same SD Card for this process.

### What You Need {docsify-ignore}

?> If youre using Tiramisu, this tutorial will still work, simply skip the Aroma download step when reading the instructions.

- The latest version of [Aroma](https://aroma.foryour.cafe)

>     - Click on `Download Tiramisu`

Reviewer: You are having the guide say to download Aroma but then the instructions for it say to "Download Tiramisu".

### What You Need {docsify-ignore}

?> If you are using Tiramisu, this tutorial will still work; simply skip the Aroma download step when reading the instructions.

- The latest version of [Aroma](https://aroma.foryour.cafe).
    - Click on `Download Payloads` and `Download Base Aroma`.
    - If you have already installed Aroma, you will not need to redownload it.
- The latest release of the [vwii-compat-installer](https://wiiu.cdn.fortheusers.org/zips/CompatTitleInstaller.zip).

- <a href="docs/files/Patched_IOS80_Installer_for_vWii.zip" download>Patched IOS 80 Installer for vWii</a>
- <a href ="docs/files/d2x_cIOS_Installer.zip" download>d2x cIOS Installer</a>

### Instructions {docsify-ignore}

1. Insert your Wii U's SD Card into your PC.
1. Copy the `apps` folder from the <code>Patched_<wbr>IOS80_<wbr>Installer_<wbr>for_<wbr>vWii<wbr>.zip</code> file to the root of your SD Card.
1. Copy the `d2x-cios-installer` from the <code>d2x_<wbr>cIOS_<wbr>Installer<wbr>.zip</code> file to the apps folder on your SD Card.
1. Copy the contents of the downloaded Tiramisu *`.zip`* file to the root of your SD Card.

> 1. Copy the `wiiu` folder to the root of your SD Card.

Reviewer: ```suggestion
1. Copy the `wiiu` folder from both Aroma `.zip` files to the root of your SD Card.

```

1. Take the SD Card out of your computer and plug it into the Wii U console.

?> Please ensure you've made a NAND Backup before continuing. If you have not, please go to [here](https://wiiu.hacks.guide/#/tiramisu/nand-backup) for instructions.

### Installing the Homebrew Channel

1. Boot into Aroma.

> 1. Launch compat_installer from the Home Menu.

Reviewer: ```suggestion
1. Launch compat_installer from the Wii U Menu.

```

1. Press `A` to install the Homebrew Channel and wait until you see `Install succeeded`, then press the HOME button to return to the Wii U Menu.

> 1. Launch your vWii (the Wii Menu icon).

Reviewer: ```suggestion
1. Launch the vWii (the Wii Menu icon).

```

   - If the installation has succeeded, you should see the Homebrew Channel installed on your Wii Menu.

### Installing cIOSes

!> Make sure you have no `.wad` files anywhere else than in the `apps` folder on your SD Card.

1. Power on your Wii U and launch the vWii.
1. Launch The Homebrew Channel.
1. Launch d2x cIOS Installer.
1. Set everything to match the following:
    - Select cIOS: `d2x-v11-beta1-vWii`
    - Select cIOS base: `56`
    - Select cIOS slot: `249`

> 1. Press the `A` button to install

Reviewer: ```suggestion
1. Press the `A` button to install.

```

1. Set everything to match the following:
    - Select cIOS: `d2x-v11-beta1-vWii`
    - Select cIOS base: `57`
    - Select cIOS slot: `250`

> 1. Press the `A` button to install

Reviewer: ```suggestion
1. Press the `A` button to install.

```

---
This page will guide you through the process of modding your vWii.

?> **Notice**
    Your SD Card will need to be formatted as FAT32. If your SD Card is not formatted to FAT32, use [GUIFormat](http://ridgecrop.co.uk/index.htm?guiformat.htm) with 32k (32768) Allocation unit size to format it. **Do not** label the SD Card as `wiiu` or it will cause issues with homebrew.
    
?> If you have hacked your Wii U in the past, you can use the same SD Card for this process.

### What You Need {docsify-ignore}

?> If youre using Tiramisu, this tutorial will still work, simply skip the Aroma download step when reading the instructions.

- The latest version of [Aroma](https://aroma.foryour.cafe)
    - Click on `Download Payloads` and `Download Base Aroma`
    - If you have already installed Aroma, you will not need to redownload it.

> - The latest release of the [vwii-compat-installer](https://apps.fortheusers.org/wiiu/CompatTitleInstaller)

Reviewer: Should probably have it be a direct download link
```suggestion
- The latest release of the [vwii-compat-installer](https://wiiu.cdn.fortheusers.org/zips/CompatTitleInstaller.zip)

```

- <a href="docs/files/Patched_IOS80_Installer_for_vWii.zip" download>Patched IOS 80 Installer for vWii</a>
- <a href ="docs/files/d2x_cIOS_Installer.zip" download>d2x cIOS Installer</a>

### Instructions {docsify-ignore}

1. Insert your Wii U's SD Card into your PC.
2. Copy the `apps` folder from the <code>Patched_<wbr>IOS80_<wbr>Installer_<wbr>for_<wbr>vWii<wbr>.zip</code> file to the root of your SD Card
3. Copy the `d2x-cios-installer` from the <code>d2x_<wbr>cIOS_<wbr>Installer<wbr>.zip</code> file to the apps folder on your SD Card
4. Copy the contents of the downloaded Tiramisu *`.zip`* file to the root of your SD Card

> 5. Copy the `compat_installer` folder to the `wiiu/apps` folder on your SD Card

Reviewer: Why not just copy the wiiu folder from the zip to the root?

Author: This is just a small fear that the Wii U folders might duplicate on certain OS's or even Windows with certain settings. It may seem simpler to just do what youre suggesting though.

Reviewer: Well as it is right now, users will have to dig through `wiiu/apps` on the zip file when there is no mention of this, along with just copying the `wiiu` folder is easy. Windows doesn't generally create duplicate folders, that is more of an issue seen on ChromeOS (and maybe macOS).

---
This page will guide you through the process of modding your vWii.

?> **Notice**
    Your SD Card will need to be formatted as FAT32. If your SD Card is not formatted to FAT32, use [GUIFormat](http://ridgecrop.co.uk/index.htm?guiformat.htm) with 32k (32768) Allocation unit size to format it. **Do not** label the SD Card as `wiiu` or it will cause issues with homebrew.
    
?> If you have hacked your Wii U in the past, you can use the same SD Card for this process.

### What You Need {docsify-ignore}

> ?> If youre using Tiramisu, this tutorial will still work, simply skip the Aroma download step when reading the instructions.

Reviewer 2: ```suggestion
?> If you are using Tiramisu, this tutorial will still work; simply skip the Aroma download step when reading the instructions.
```

### Installing the Homebrew Channel

1. Boot into Aroma
1. Launch compat_installer from the Home Menu
1. Press `A` to install the Homebrew Channel and wait until you see `Install succeeded`, then press the HOME button to return to the Wii U Menu
1. Launch your vWii (the Wii Menu icon)
   - If the installation has succeeded, you should see the Homebrew Channel installed on your Wii Menu

### Installing cIOSes

!> Make sure you have no `.wad` files anywhere else than in the `apps` folder on your SD Card.

1. Power on your Wii U and launch the vWii
1. Launch The Homebrew Channel
1. Launch d2x cIOS Installer
1. Set everything to match the following:
    - Select cIOS: `d2x-v11-beta1-vWii`
    - Select cIOS base: `56`
    - Select cIOS slot: `249`
1. Press the `A` button to install
1. Set everything to match the following:
    - Select cIOS: `d2x-v11-beta1-vWii`
    - Select cIOS base: `57`
    - Select cIOS slot: `250`
1. Press the `A` button to install
1. Set everything to match the following:
    - Select cIOS: `d2x-v11-beta1-vWii`
    - Select cIOS base: `58`
    - Select cIOS slot: `251`
1. Press the `A` button to install
1. Press the `B` button to exit

### Patching IOS 80

> !> If for any reason the Wii U is turned off while patching IOS 80, your vWii will be bricked. This can be fixed by either extracting the IOS 80 from your previously made NAND backup and then replacing it over FTP, or by [reinstalling IOS 80](recover-vwii-ioses-channels).

Reviewer 2: ```suggestion
!> If, for any reason, the Wii U is turned off while patching IOS 80, your vWii will be bricked. This can be fixed by either extracting the IOS 80 from your previously made NAND backup and then replacing it over FTP, or by [reinstalling IOS 80](recover-vwii-ioses-channels).
```

1. Launch Patched IOS 80 Installer for vWii
1. Read the warning screen and wait 30 seconds
1. Press any buttons to install
1. Wait until it says <code>IOS80 <wbr>Installation <wbr>is <wbr>complete!</code>
1. Press any buttons to exit

> !> Priiloader versions before 0.10.0 do not work with vWii. Installing versions of Priiloader before 0.8.2 will brick your vWii.

Reviewer 2: These two sentences seem to imply the exact same thing. Could it be a bit more concise? 

Such as
`Do not install any Priiloader version prior to 0.10.0, or it will brick your vWii.`
?

Reviewer: I worded it that way specifically since 0.8.2-0.9.1 would refuse to install vwii, while anything before 0.8.2 had zero checks and would install priiloader, then bricking vwii.

Reviewer 2: I see. We can leave it alone in that case.

> 1. Copy the `compat_installer.elf` file to the `wiiu/apps` folder on the root of your SD Card.

Reviewer: Wouldn't it make sense to use the rpx port which works on both Aroma *and* Tiramisu? `.elf` is not supported on Aroma at all.

1. Take the SD Card out of your computer and plug it into your Wii U console.

> ?> Please ensure youve made a NAND Backup before continuing. If you have not, please go to [here]( https://wiiu.hacks.guide/#/tiramisu/nand-backup) for instructions.

Reviewer: I don't see the need to link to the main guide backup page, also multiple misspellings

Author: Because having another NAND Guide within the vWii section is kinda redundant when a NAND Backup page already exists.
Also, where are the misspellings exactly? Id love for them to be pointed out so i can fix them.

Reviewer: `youve`
`[here]( https://wiiu.hacks.guide/#/tiramisu/nand-backup)` should not have a space

Author: One word doesnt really mean "multiple" mispellings, i did fix that with the latest commit i did. Also, that space was a mistake i didnt notice, oops

### Installing the Homebrew Channel

1. Boot into Aroma

> 1. Launch the Homebrew Launcher, by opening the Mii Maker

Reviewer: Not sure how you can boot into Aroma and then launch HBL when HBL does not exist on Aroma

1. Launch the compat_installer
1. Press `A` to install the Homebrew Channel and wait until you see `Install succeeded`. Then press the HOME button to return to the Wii U Menu
1. Launch vWii (the Wii Menu icon)
   - If the installation has succeeded, you should see the Homebrew Channel in your Wii Menu

### Installing cIOSes

!> Make sure you have no `.wad` files anywhere else than in the `apps` folder on your SD Card.

1. Power on your Wii U and launch the vWii
1. Launch The Homebrew Channel
1. Launch d2x cIOS Installer
1. Set everything to match the following:
    - Select cIOS: `d2x-v11-beta1-vWii`
    - Select cIOS base: `56`
    - Select cIOS slot: `249`
1. Press the `A` button to install
1. Set everything to match the following:
    - Select cIOS: `d2x-v11-beta1-vWii`
    - Select cIOS base: `57`
    - Select cIOS slot: `250`
1. Press the `A` button to install
1. Set everything to match the following:
    - Select cIOS: `d2x-v11-beta1-vWii`
    - Select cIOS base: `58`
    - Select cIOS slot: `251`
1. Press the `A` button to install
1. Press the `B` button to exit

### Patching IOS 80

> !> If for any reason the Wii U is turned off while patching IOS 80, your vWii will be bricked. This can be fixed by either extracting the IOS 80 from your previously made NAND backup and then replacing it over FTP, using [vWii Decaffeinator](https://github.com/GaryOderNichts/vWii-Decaffeinator), or by [reinstalling IOS 80](recover-vwii-ioses-channels).

Reviewer: There is no need to link decaffeinator if the guide linked at "reinstalling IOS 80" already has you use decaffeinator.

Author: This couldve been more clear, because decaffeinator was not linked when the "reinstalling IOS 80" text was there. I simply misjudged what it mean sadly.
